packages/payload/src/database/migrations/getMigrations.ts

```diff
@@ -12,7 +12,7 @@ export async function getMigrations({
   const migrationQuery = await payload.find({
     collection: 'payload-migrations',
     limit: 0,
-    sort: '-name',
+    sort: '-batch',
```

Comment:
We should be able to sort on `-name` and have a consistent result. If it is only sorted on `batch` that could show them in the wrong order. In some cases all your migrations might have been run in the same batch.

Reply:
This fixes another test - https://github.com/payloadcms/payload/pull/9330/files#diff-2a01d79fb3010bfe668ca296a4b7e534f28386e38f8fa1fa379dd309ed0f67c1R259

I refactored this test to actually do what it should:
```ts
// migrate existing if there any (in beforeEach we create a migration)
await payload.db.migrate()

await payload.db.createMigration({
  forceAcceptWarning: true,
  migrationName: 'migration_to_down',
  payload,
})

// migrate current to test
await payload.db.migrate()

const { docs } = await payload.find({ collection: 'payload-migrations' })
// should be migration
expect(docs.some((doc) => doc.name.includes('migration_to_down'))).toBeTruthy()

await payload.db.migrateDown()

const migrations = await payload.find({
  collection: 'payload-migrations',
})

// should delete this migration
expect(migrations.docs.some((doc) => doc.name.includes('migration_to_down'))).toBeFalsy()
```

But now I understood, good find. Still, could it be the case for multiple sort? `-batch,-name`, seems to work.

Reply:
Example when this migrations test flakes https://github.com/payloadcms/payload/actions/runs/12012820172/job/33485183442

Reply:
I thought about the mutli-sort option also. That makes the most sense to me, though I still don't get how the migration name sort would not have the expected order based on how we generate these names.